classes/Cluster.py

```diff
@@ -118,11 +118,13 @@ def print_all_scooters(self, with_coordinates=False):
             )
         return string if string != "" else "Empty cluster"
 
-    def get_swappable_scooters(self):
+    def get_swappable_scooters(self, battery_limit=70):
```

Comment:
Where is this function used? Can the default battery limit of 70 ( changed from 100) create any issues in other functions?

Reply:
Should be ok for the other policies and for behavior of the model